### DISCUSSION TITLE
Adding SSO to the Bot Composer Project

### DISCUSSION THREAD

**Discussion by jacobpoly73** [microsoft/BotFramework-Composer#8982] [Q&A]
Are there any documented steps to add SSO to the Bot Composer project? Essentially I want to block the bot from unauthorized access. I think the way to do this is to add either SSO or a Login step. That way, without a tenant AAD account, the user is unable to interact with the Bot. Any suggestions on how to approach this?

**Comment by Pat-On** [comment]
Hey @jacobpoly73, Did You find the solution to Your problem? I am looking for a way of connecting the Qna/Luis orchestrator bot with AAD authorisation and I could not find anything either.... So that is why I am curious if You solved Your problem.
